src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,69 @@
+package duke;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+import duke.task.Task;
+
+/**
+ * Used to store to or load from the task lists into the disk
+ */
+public class Storage {
+
+    private Path filePath;
+
+    /**
+     * Creates a storage corresponding to the path.
+     *
+     * @param path the location of file in the disk.
+     */
+    public Storage(String path) {
+        this.filePath = Paths.get(path);
+    }
+
+    /**
+     * Loads the task list file from filePath.
+     *
+     * @return a list of task corresponding to the file.
+     * @throws IOException if such filePath does not exist.
+     */
+    public List<Task> load() throws IOException {
```

Comment:
I feel the load method could be named more specific such as defining it as `loadTasks`, in case if the Storage method has multiple load commands in the future. Just my thoughts :)